recipes/jsonrpcserver/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set name = "jsonrpcserver" %}
+{% set version = "4.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 56f3ab21aad908920538ab32e0a608c1d39ccb0e00c7a112f13484e53b434542
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Could it be ``noarch``?
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```

---

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

---

Added noarch. Thanks for reviewing :)